test/compiler/irpasses.jl

```diff
@@ -4,7 +4,7 @@ using Test
 using Base.Meta
 import Core:
```

Comment:
```suggestion
using Core.IR
```
would be cleaner.

Reply:
Hmm, I really appreciate explicit imports, personally.

Easily >50% of my confusion reading new Julia code is figuring out where a name came from, but in this case it's pretty obvious so I'm happy to make the change.